admin/tests/integration/components/certification-center-memberships-section-test.js

```diff
@@ -0,0 +1,54 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import EmberObject from '@ember/object';
+
+module('Integration | Component | certification-center-memberships-section', function(hooks) {
+
+  setupRenderingTest(hooks);
+
+  test('it should display certification center memberships details', async function(assert) {
```

Comment:
**chipotage**
Il n'y a qu'un membership ici, donc je propose d'enlever le `s`
`  test('it should display certification center memberships detail', async function(assert) {`

Reply:
Done